Woof/Woof/Model/Walking/Walking+Dummy.swift

```diff
@@ -0,0 +1,75 @@
+import Foundation
+
+extension Walking {
+    /// Provides dummy data for `Walking` model.
+    enum Dummy {
+        static let dummyWalking = Walking(
```

Comment:
Missing doc

Reply:
In my opinion, you could understand the purpose of these constants from their names.
Fixed with 7f70e76